src/repositories/schemas.ts

```diff
@@ -10,14 +10,17 @@ export interface Vote {
 
 export interface Ballot {
   _id: string;
+  createdBy: string;
   running: boolean;
+  question: string;
   options: VotingOption[];
 }
 
 export interface Token {
   _id: string;
   token: string;
   valid: boolean;
+  createdBy: string;
```

Comment:
Why are we saving the user who created the ballot/token ?

Reply:
Useful for analysis. Especially if someone gets access who should not have any, we can invalidate all tokens created by him, for example.